src/desktop/auth-client.desktop

```diff
@@ -7,21 +7,15 @@ X-KDE-HasReadOnlyMode=true
 X-SuSE-YaST-Call=auth-client
 
 X-SuSE-YaST-Group=Net_advanced
-X-SuSE-YaST-Argument=
 X-SuSE-YaST-RootOnly=true
 X-SuSE-YaST-AutoInst=all
-X-SuSE-YaST-Geometry=
-X-SuSE-YaST-SortKey=
 X-SuSE-YaST-AutoInstResource=auth-client
 X-SuSE-YaST-AutoInstClonable=true
-X-SuSE-YaST-AutoInstRequires=
-X-SuSE-YaST-AutoInstSchema=auth-client.rnc
-X-SuSE-YaST-Keywords=authentication,sssd,pam,ldap,krb5
+X-SuSE-YaST-Keywords=authentication,sssd,pam,ldap,kerberos,krb,krb5,windows,microsoft,ad,activedirectory
 
 Icon=yast-kerberos
 Exec=xdg-su -c "/sbin/yast2 auth-client"
 
-Name=Authentication Client
-GenericName=Configure the authentication client (sssd)
+Name=User Logon Management
```

Comment:
Why not add a title that repeats the icon name used to launch it, like every other module does? See the title parameter in http://www.rubydoc.info/github/yast/yast-yast2/Yast/WizardClass:SetContents